inst/extdata/mvp/product-carbon-transition-risk_review.Rmd

```diff
@@ -0,0 +1,308 @@
+production stage of the product and not the emissions from the inputs.
+The unit is CO2e in kg.
+
+The goal of the transition risk MVP is to create a first draft of how
+the transition risk indicator for products would be build in code in
+order to convert it into code production easily in the future. This MVP
+should provide the share of products with “low”, “medium”, or “high”
+relative production emissions per company.
+
+In this markdown file the aim is to prepare data so that they contain
+the following:
+
+- a column with production emissions
+- a column indicating the percentile relative to (i) all products with
+  same unit (ii) products in same sector (iii) products in same segment
+- a column indicating whether the product has “low”, “medium” or “high”
+  relative production emissions
+
+The product carbon transition risk indicator consists of 2 main steps:
+
+    - Step 1: Calculating the relative carbon footprint per product
+    - Step 2: Aggregating on the company-level
+
+### Step 1: Calculating the relative carbon footprint per product
+
+## Data
+
+## Prepare data so that they contain the respective columns:
+
+- replace sample activities for companies with “real activities” from
+  Hanish’s sample
+- include absolute score
+- use ecoinvent’s v3.9 data
+
+## 1. Load packages and ecoinvent data
+
+``` {r}
+library(rlang)
+library(fs)
+library(tidyr)
+library(readr)
+library(dplyr)
+library(graphics)
+library(tiltIndicator)
+
+set.seed(123)
+
+data_ecoinvent <- params$input1 %||% 
+    read_csv(pctr_path("input_data", "ecoinvent_activities.csv"))
+```
+
+## 2. Create subset of data with relevant columns
+
+``` {r}
+data_short <- data_ecoinvent %>%
+  select(all_of(c(
+    "Activity Name",
+    "Geography",
+    "Time Period",
+    "Special Activity Type",
+    "Sector",
+    "ISIC Classification",
+    "ISIC Section",
+    "Reference Product Name",
+    "CPC Classification",
+    "Unit"
+  )))
+
+# renaming
+names(data_short)[names(data_short) == "Activity Name"] <- "activity"
+names(data_short)[names(data_short) == "Geography"] <- "geo"
+names(data_short)[names(data_short) == "Time Period"] <- "time"
+names(data_short)[names(data_short) == "Special Activity Type"] <- "special_act_type"
+names(data_short)[names(data_short) == "Sector"] <- "sec"
+names(data_short)[names(data_short) == "ISIC Classification"] <- "isic_class"
+names(data_short)[names(data_short) == "ISIC Section"] <- "isic_sec"
+names(data_short)[names(data_short) == "Reference Product Name"] <- "ref_prod"
+names(data_short)[names(data_short) == "CPC Classification"] <- "cpc_class"
+names(data_short)[names(data_short) == "Unit"] <- "unit"
+```
+
+## 3. Add column “production_emissions_per_product”
+
+``` {r}
+# create column with sample emission data, using "rnorm" with normal distribution
+values <- rnorm(nrow(data_short), mean = 6, sd = 1.5)
+data_short <- data_short %>%
+  mutate(production_emissions = values, emissions_unit = "per product unit")
```

Comment:
Do this outside this main proces and add the pre-fabricated column already into the demo data. To keep a record of how you did it, you can keep the script in data-raw/name-of-resulting-clean-dataset.R.